timelock-impl/src/test/java/com/palantir/atlasdb/timelock/management/TimeLockManagementResourceTest.java

```diff
@@ -113,6 +113,7 @@ public void setup() throws MalformedURLException {
                 allNodesDisabledNamespacesUpdater,
                 authHeaderValidator,
                 redirectRetryTargeter,
+                0.0,
                 new ServiceLifecycleController(serviceStopper, PTExecutors.newSingleThreadScheduledExecutor()));
```

Comment:
The newSingleThreadScheduledExecutor thread will run forever, should be shut down after the test